packages/contracts-bedrock/scripts/ChainAssertions.sol

```diff
@@ -140,11 +140,23 @@ library ChainAssertions {
     }
 
     /// @notice Asserts that the OptimismMintableERC20Factory is setup correctly
-    function checkOptimismMintableERC20Factory(Types.ContractSet memory _contracts) internal view {
+    function checkOptimismMintableERC20Factory(
+        Types.ContractSet memory _contracts,
+        bool _isInitialized
+    )
+        internal
+        view
+    {
         console.log("Running chain assertions on the OptimismMintableERC20Factory");
         OptimismMintableERC20Factory factory = OptimismMintableERC20Factory(_contracts.OptimismMintableERC20Factory);
-        require(factory.BRIDGE() == _contracts.L1StandardBridge);
-        require(factory.bridge() == _contracts.L1StandardBridge);
+
```

Comment:
we could add a check that `vm.load(address(factory, bytes32(0)) == bytes32(uint(1))` to verify that its initialized. This would apply in both the proxy/impl case.